src/validator/GoValidator.cpp

```diff
@@ -798,5 +797,11 @@ Status GoValidator::buildColumns() {
     return Status::OK();
 }
 
+bool GoValidator::notAllPropsEmpty() {
+    return !exprProps_.inputProps().empty() || !exprProps_.varProps().empty() ||
+        !exprProps_.srcTagProps().empty() || !exprProps_.edgeProps().empty() ||
+        !exprProps_.dstTagProps().empty() || from_.fromType != FromType::kInstantExpr;
+}
+
```

Comment:
```
!exprProps_.inputProps().empty() || !exprProps_.varProps().empty() ||
        !exprProps_.srcTagProps().empty() || !exprProps_.edgeProps().empty() ||
        !exprProps_.dstTagProps().empty()
```
Would it be better move this into ExprProps?
Then the logic in line 757 could be 
```
if (exprProps_.notAllPropsEmpty() || from_.fromType != FromType::kInstantExpr)
```